contracts/interchain-token-service/src/contract.rs

```diff
@@ -198,7 +205,16 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
             }
         }
 
-        Ok((deployed_address, token_id))
+        let token_data = TokenData {
+            token_address: deployed_address,
+            token_manager_type: TokenManagerType::NativeInterchainToken,
+        };
+
+        env.storage()
+            .persistent()
+            .set(&DataKey::TokenId(token_id.clone()), &token_data);
```

Comment:
create a helper to do this since it'll be used in other places too

Reply:
make sense, updated.